### PR TITLE
Add publish-to-automation-hub-3pci

### PR DESCRIPTION
**Pull request by alinabuzachis**
Add `publish-to-automation-hub-3pci` to be used by `community.okd `

### CODE REVIEW

**Review by goneri — APPROVED** [review]

**Comment by pabelanger** [comment]
@alinabuzachis @goneri we don't upload community collections to automation hub, so curious why we are doing this?

File: zuul.d/project-templates.yaml [hunk 1]
```diff
@@ -1392,6 +1392,25 @@
               - name: release-ansible-collection-galaxy
                 soft: true
 
+- project-template:
+    name: publish-to-automation-hub-3pci
+    description: |
+      Publish an Ansible collection to Ansible Automation Hub
+    third-party-check:
```
Comment (pabelanger) [inline zuul.d/project-templates.yaml:1399]: this is the wrong pipeline for 3pci-check

**Comment by alinabuzachis** [comment]
> @alinabuzachis @goneri we don't upload community collections to automation hub, so curious why we are doing this?

`redhat.openshift` is cut from `community.okd` and we publish it to the automation hub. @goneri

**Comment by pabelanger** [comment]
@alinabuzachis how does that work? If you were to tag community.okd today, it would build that tarball and publish it to automation hub.  Where does the redhat.openshift changes take place?

File: zuul.d/project-templates.yaml (hunk 1 above)
Comment (goneri) [reply zuul.d/project-templates.yaml:1399]: oh indeed. Let's fix that.

Comment (goneri) [reply zuul.d/project-templates.yaml:1399]: https://github.com/ansible/ansible-zuul-jobs/pull/1124

**Comment by pabelanger** [comment]
what we should do, is the same thing we do for vmware modules. i believe @goneri will sync commits between the 2 projects, which means 2 repos.  I don't think we have automation that will take the community repo directly and rename it to a redhat collection.

**Comment by goneri** [comment]
> > @alinabuzachis @goneri we don't upload community collections to automation hub, so curious why we are doing this?
> 
> @pabelanger `redhat.openshift` is cut from `community.okd` and we publish it to the automation hub. @goneri

Oh! Yes, you're right. I totally forgot about.

**Comment by goneri** [comment]
> > > @alinabuzachis @goneri we don't upload community collections to automation hub, so curious why we are doing this?
> > 
> > 
> > @pabelanger `redhat.openshift` is cut from `community.okd` and we publish it to the automation hub. @goneri
> 
> Oh! Yes, you're right. I totally forgot about.

-> https://github.com/ansible/project-config/pull/947

**Pull request merged**